src/Azure.php

```diff
@@ -98,7 +99,7 @@ protected function handleTesting(Request $request, Closure $next)
      */
     public function getAzureUrl()
     {
-        return $this->baseUrl . config('azure.tenant_id') . $this->route2 . "authorize?response_type=code&client_id=" . config('azure.client.id') . "&domain_hint=" . urlencode(config('azure.domain_hint')) . "&scope=" . urldecode(config('azure.scope'));
+        return $this->baseUrl . config('azure.tenant_id') . $this->route2 . "authorize?response_type=code&client_id=" . config('azure.client.id') . "&domain_hint=" . urlencode(config('azure.domain_hint')) . "&scope=" . urldecode(config('azure.scope'))  . '&redirect_uri=' . urlencode(route('azure.callback'));
```

Comment:
Let's create a string called `url` and then set it to what it was originally.  Then, check to see if the `azure.callback` route exists and if it does, append as suggested.